core/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          xmlns:tools="http://schemas.android.com/tools" package="de.danoeh.antennapod.core">
+    xmlns:tools="http://schemas.android.com/tools" package="de.danoeh.antennapod.core">
```

Comment:
Please revert unrelated change

Reply:
I think this may have been an auto-formatting issue.